.travis.yml

```diff
@@ -1,26 +1,32 @@
 language: scala
-scala: 2.12.3
+scala: 2.12.4
 jdk: oraclejdk8
 
-script: ./bin/runci.sh "$CI_TEST"
 env:
   global:
     - TRAVIS_NODE_VERSION="8"
     - TRAVIS_YARN_VERSION="1.1.0"
-  matrix:
-    - CI_TEST: ci-test
-      CI_PUBLISH: true
-    - CI_TEST: scripted
-    - CI_TEST: scalafmt
+
+matrix:
```

Comment:
👍 This is a better way to organize .travis.yml

Reply:
Yes as long as the different builds are one liners, although I had to define some potentially useless env vars to have nice build names in Travis.

Reply:
The env vars make the travis matrix a lot easier to read.